CODE_OF_CONDUCT.md

```diff
@@ -44,23 +44,24 @@ Everyone who participates in the Turing Way project activities is required to co
-* **Be considerate of others’ participation**. Everyone should have an opportunity to be heard. In update sessions, please keep comments succinct so as to allow maximum engagement by all participants. Do not interrupt others on the basis of disagreement; hold such comments until they have finished speaking.
-* **Don’t be a bystander**. If you see something inappropriate happening, speak up. If you don't feel comfortable intervening but feel someone should, please feel free to ask a member of the Code of Conduct response team for support.
-* As an overriding general rule, please **be intentional in your actions and humble in your mistakes**.
+- Be respectful of different viewpoints and experiences. Do not engage in homophobic, racist, transphobic, ageist, ableist, sexist, or otherwise exclusionary behaviour.
+- Use welcoming and inclusive language. Exclusionary comments or jokes, threats or violent language are not acceptable. Do not address others in an angry, intimidating, or demeaning manner. Be considerate of the ways the words you choose may impact others. Be patient and respectful of the fact that English is a second (or third or fourth!) language for some participants.
+- Do not harass people. Harassment includes unwanted physical contact, sexual attention, or repeated social contact. Know that consent is explicit, conscious and continuous—not implied. If you are unsure whether your behaviour towards another person is welcome, ask them. If someone tells you to stop, do so.
+- Respect the privacy and safety of others. Do not take photographs of others without their permission. Do not share other participants' personal experiences without their express permission. Note that posting (or threatening to post) personally identifying information of others without their consent ("doxing") is a form of harassment.
+- Be considerate of others' participation. Everyone should have an opportunity to be heard. In update sessions, please keep comments succinct so as to allow maximum engagement by all participants. Do not interrupt others on the basis of disagreement; hold such comments until they have finished speaking.
+- Don't be a bystander. If you see something inappropriate happening, speak up. If you don't feel comfortable intervening but feel someone should, please feel free to ask a member of the Code of Conduct response team for support.
+- Keep conversations open and inclusive. Avoid direct messaging (DMs) when discussing project-related topics, as this can limit collaboration and transparency. Instead, use public channels or threads to encourage participation and maintain open communication among community members.
```

Comment:
```suggestion
- Keep conversations open and inclusive. While some communications may require it, aim to avoid direct messaging (DMs) when discussing project-related topics, as this can limit collaboration and transparency with others. Instead, use public channels or threads to encourage participation and maintain open communication among community members.
```
Softens language a bit - what do you think? Too much?